File: app/src/main/java/com/neome/api/ent/entDrawer/sig/SigEntAvatarUser.java
// neome.ai API - do not change
//

package com.neome.api.ent.entDrawer.sig;

import com.neome.api.meta.base.Types.EntId;
import com.neome.api.meta.base.Types.MediaIdAvatar;
import com.neome.api.nucleus.base.sig.SigVersion;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class SigEntAvatarUser extends SigVersion
{
  @Nullable
  public String about;

  @Nullable
  public MediaIdAvatar avatarId;

  public EntId entId;

  public String name;
}

File: app/src/main/java/com/neome/api/ent/entDrawer/sig/SigEntAvatarUser.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.entDrawer.sig

import com.neome.api.meta.base.Types.EntId
import com.neome.api.meta.base.Types.MediaIdAvatar
import com.neome.api.nucleus.base.sig.SigVersion

interface SigEntAvatarUser : SigVersion
{
  val about: String?
  val avatarId: MediaIdAvatar?
  val entId: EntId
  val name: String
}